### PR TITLE
main/lz4: upgrade to 1.9.1

### PR DESCRIPTION
**Pull request by itoffshore**
* https://github.com/lz4/lz4/blob/dev/NEWS

### CODE REVIEW

File: main/lz4/APKBUILD [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-pkgver=1.8.3
-pkgrel=2
+pkgver=1.9.0
+pkgrel=0
 pkgdesc="LZ4 is lossless compression algorithm with fast decoder @ multiple GB/s per core."
 url="https://github.com/lz4/lz4"
 arch="all"
 license="BSD-2-Clause GPL-2.0-only"
 checkdepends="diffutils"
+options="!check" # disabled due to incompatibility with drone https://github.com/lz4/lz4/issues/675
```
Comment (bratkartoffel) [inline main/lz4/APKBUILD]: If the issue is specific with drone we shouldn't disable the tests

Comment (itoffshore) [reply main/lz4/APKBUILD]: The tests pass in my development environment & [under `travis`](https://travis-ci.org/alpinelinux/aports/builds/521363338?utm_source=github_status&utm_medium=notification).

If this `PR` can be pulled with `drone` failing I will change it back

Comment (itoffshore) [reply main/lz4/APKBUILD]: fixed in https://github.com/lz4/lz4/pull/686 & applied in `fix-drone-tests.patch`

**Comment by itoffshore** [comment]
rebased to `1.9.1`

**Comment by tcely** [comment]
@clandmeter is the hold label still appropriate?

**Comment by maxice8** [comment]
ping

**Pull request closed without merging**